# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/luit/hotfix-def.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- luit-1.1.1.orig/sys.c	2012-03-23 05:23:07.000000000 +0100
+++ luit-1.1.1/sys.c	2026-07-26 06:34:54.775075583 +0200
@@ -20,6 +20,8 @@
 THE SOFTWARE.
 */
 
+#define _GNU_SOURCE
+
 #ifdef HAVE_CONFIG_H
 # include "config.h"
 #endif
